hack/ocm-minikube.sh

```diff
@@ -944,16 +944,18 @@ hub_kubeconfig_file_create()
 	kubectl --context $hub_cluster_name config view --flatten --minify >$hub_kubeconfig_file_path_name
 }
 exit_stack_push unset -f hub_kubeconfig_file_create
-deploy()
+ocm_minikube_deploy()
 {
+	hub_cluster_name=$1
+	spoke_cluster_names=$2
```

Comment:
This creates a globals - is this used to pass the arguments to the functions below?

This is not a good direction - this code is has enough issues without globals.